<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Tasky</class>
 <widget class="QMainWindow" name="Tasky">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1023</width>
    <height>485</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tasky</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Agregar Tarea</string>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>9</number>
          </property>
          <property name="rightMargin">
           <number>9</number>
          </property>
          <property name="bottomMargin">
           <number>9</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Tarea</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="editTarea">
            <property name="text">
             <string/>
            </property>
            <property name="placeholderText">
             <string>Ingrese una tarea</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Asignatura:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QComboBox" name="cmbAsignatura">
            <item>
             <property name="text">
              <string>Calculo</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Programacion Orientada a Objetos</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Fisica para Ciencias de la Computacion</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Electrotecnia</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Estructura de Datos</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Ã‰tica</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Fecha: </string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QDateEdit" name="dtFecha">
            <property name="dateTime">
             <datetime>
              <hour>0</hour>
              <minute>0</minute>
              <second>0</second>
              <year>2023</year>
              <month>12</month>
              <day>19</day>
             </datetime>
            </property>
            <property name="minimumDate">
             <date>
              <year>2023</year>
              <month>12</month>
              <day>19</day>
             </date>
            </property>
            <property name="displayFormat">
             <string>d/M/yyyy</string>
            </property>
            <property name="calendarPopup">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Hora:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QTimeEdit" name="dtHora">
            <property name="calendarPopup">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QWidget" name="widget_2" native="true">
            <layout class="QFormLayout" name="formLayout_2">
             <property name="horizontalSpacing">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item row="0" column="0">
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>268</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item row="0" column="1">
              <widget class="QPushButton" name="btnAgregar">
               <property name="text">
                <string>Agregar</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Lista de Tareas</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QTableWidget" name="tblTareas"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1023</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
